### ISSUE TITLE
This BrainKey is not compatible with BTS 1.0 message in create account

### ISSUE THREAD

**Issue by svk31** [cryptonomex/graphene-ui#840]
![image](https://cloud.githubusercontent.com/assets/6890015/14976485/5c4b51de-110c-11e6-8d70-56911241995f.png)

Always visible

**Comment by xeroc** [comment]
I don't even understand that message. If you provide brain wallet from BTS1, it should be compatible, if you provide it from somewhere else, people don't need to care.

The only scenario is that if people provide a brainkey that is supposed to be BTS1 compatible, but isn't. not sure if that is even possible unless they have a typo.

The import of brainwallets could be improved quite a lot by spell checking against the brain-key dictionary

**Comment by YazenAmind** [comment]
I have the same problem. Also, if I want to send obits or publish anything, I get transaction did not occur do you have a private key (in red).  I have user name, password, brainkey, memo key and Icannot use my account or do anything although I can see the balance. really frustrating.

**Comment by svk31** [comment]
This text was removed in https://github.com/cryptonomex/graphene-ui/commit/22548e5061a40bf674df8ccd17526d4545ddb7c4.

@YazenAmind your issue is unrelated, sounds to me like the wallet you're using does not have the private keys of the account, perhaps you've cleared your cache and removed them. Try restoring the wallet from a backup or using the brainkey.

**Comment by YazenAmind** [comment]
Thanks - the only reason I replied was I had the same exact reply as the original author. I did not clean the cache and I have tried restoring the back up. My problem is I bought the obits prior to a wallet being created on Feb 24, left town and didn't access until. Mid march when the same address I used was being stopped as insecure.  My mistake is probably going through the same process and creating another account name. I do appreciate you taking the time to try to help. I tried the brainkey and was told not compatible with bts 1.0
All the best
Yazen
Sent on the Sprint® Now Network from my BlackBerry®

-----Original Message-----
